blob_server/models.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy import (
+    Column,
+    DateTime,
+    ForeignKey,
+    func,
+    Integer,
+    String,
+    Table,
+    Text
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+base = declarative_base()
+
+
+PostTagLinks = Table(
+    'post_tag_links',
+    Base.metadata,
+    Column('post_id', Integer, ForeignKey(blog_posts.id)),
+    Column('tag_id', Integer, ForeignKey(tags.id))
+)
+
+
+class BlogPost(Base):
+    """An entry containing the created date and the content of text"""
+    __tablename__ = 'blog_posts'
+
+    id = Column(Integer, primary_key=True)
+    created = Column(DateTime(timezone=True), server_default=func.now())
+    content = Column(Text)
+    tags = relationship(
+        'Tag',
+        back_populates='blog_posts',
```

Comment:
Your `back_populates` has to match the attribute name in the `Tag` class, in this case it should just be `posts`